src/integrations/sdkinfo.ts

```diff
@@ -37,19 +48,18 @@ export class SdkInfo implements Integration {
       }
 
       event.platform = event.platform || 'javascript';
-      event.sdk = {
-        ...event.sdk,
-        name: SDK_NAME,
-        packages: [
-          ...((event.sdk && event.sdk.packages) || []),
-          ...((this._nativeSdkInfo && [this._nativeSdkInfo]) || []),
-          {
-            name: 'npm:@sentry/capacitor',
-            version: SDK_VERSION,
-          },
-        ],
-        version: SDK_VERSION,
-      };
+      event.sdk = event.sdk || {};
+      event.sdk.name = event.sdk.name || defaultSdkInfo.name;
+      event.sdk.version = event.sdk.version || defaultSdkInfo.version;
+      event.sdk.packages = [
+        // default packages are added by baseclient and should not be added here
+        ...(event.sdk.packages || []),
+        ...((this._nativeSdkPackage && [this._nativeSdkPackage]) || []),
+        {
+          name: 'npm:@sentry/capacitor',
+          version: SDK_VERSION,
+        },
```

Comment:
Since we are not adding this from the `defaultSdkInfo`, maybe we can delete that and use the SDK_NAME, SDK_PACKAGE_NAME, SDK_VERSION directly here.